indico/client/client.py

```diff
@@ -22,8 +39,18 @@ def _handle_request_chain(self, chain: RequestChain):
 
     def call(self, request: Union[HTTPRequest, RequestChain]):
         """
-        Make a call to the indico IPA platform
+        Make a call to the Indico IPA Platform
+
+        Args:
+            request (HTTPRequest or RequestChain): GraphQL request to send to the Indico Platform
```

Comment:
Graphql or HTTPRequest to make to the platform

Reply:
Changed HTTPRequest to GraphQLRequest